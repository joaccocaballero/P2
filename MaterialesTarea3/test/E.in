insertarPalabra mas
buscarFinPrefijo pa
buscarFinPrefijo ma
insertarPalabra otras
insertarPalabra palas
insertarPalabra palabra
insertarPalabra palabra
imprimirPalabrasCortas 2
imprimirPalabrasCortas 7
Fin