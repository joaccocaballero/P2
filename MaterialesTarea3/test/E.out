1>Insertando mas.
2>
palabras no contiene pa.
3>
mas
4>Insertando otras.
5>Insertando palas.
6>Insertando palabra.
7>Insertando palabra.
8>
9>
mas
otras
palabra
palas
10>Fin.
